ee/tabby-ui/lib/stores/exp-flag-store.ts

```diff
@@ -0,0 +1,12 @@
+import { createStoreWithHydrated } from './utils'
+
+interface InitialState {
```

Comment:
Maybe it could be renamed to something like 'ExperimentalFlagState'?

Reply:
yeah should rename